### ISSUE TITLE
(WCAG 2.1) Remove aria-hidden attribute from CiviForm logo

### ISSUE THREAD

**Issue by jeanniefu** [civiform/civiform#3569]
Required to meet WCAG 2.1 guidelines. Identified during an accessibility audit against WCAG 2.1.

**WCAG SC**: [1.3.2. Meaningful Sequence](https://www.w3.org/TR/WCAG21/#meaningful-sequence)

**Observed results**: The "City of CiviForm" logo is hidden (see screenshots) on login page and program list page. Logos are usually not considered decorative and should be rendered by screenreader using its alt text. An exception is when the logo has actual textual elements adjacent to it that provides the context. That is not the case here since it is just a logo with text in it and that text needs to be conveyed by screen reader.

**How to fix**: Remove `aria-hidden=true`. Note to self: check how this is set in the code since it's configurable per deployment. Do we always have the alt text set?

![image](https://user-images.githubusercontent.com/9094240/192912802-7b0d3387-63a5-4a4b-84b1-5574e2996fd3.png)
![image](https://user-images.githubusercontent.com/9094240/192914473-1cde7b9c-27e1-4330-ac19-5259e3a3f621.png)
